packages/internals/src/sendPanic.ts

```diff
@@ -31,23 +43,11 @@ export async function sendPanic(error: RustPanic, cliVersion: string, engineVers
 
     const maskedSchema: string | undefined = schema ? maskSchema(schema) : undefined
 
-    let sqlDump: string | undefined
     let dbVersion: string | undefined
     // For a SQLite datasource like `url = "file:dev.db"` only schema will be defined
     const schemaOrUrl = schema || error.introspectionUrl
-    if (error.area === ErrorArea.INTROSPECTION_CLI && schemaOrUrl) {
-      let engine: undefined | IntrospectionEngine
-      try {
-        engine = new IntrospectionEngine()
-        sqlDump = await engine.getDatabaseDescription(schemaOrUrl)
-        dbVersion = await engine.getDatabaseVersion(schemaOrUrl)
-        engine.stop()
-      } catch (e) {
-        debug(e)
-        if (engine && engine.isRunning) {
-          engine.stop()
-        }
-      }
+    if (error.area === ErrorArea.LIFT_CLI && schemaOrUrl) {
```

Comment:
The `MigrateEngine` may throw Rust panics with the `LIFT_CLI` error area, rather than the previous `INTROSPECTION_CLI` error area thrown by the `IntrospectionEngine`

Reply:
No chance of this now triggered for ME errors as well? (probably undertested area)

Reply:
Ah, I see this is only to get the database version in a safe way - not more.